src/main/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepo.scala

```diff
@@ -73,13 +73,8 @@ class RoutesPostgresRepo @Inject() (
   override def selectAll: DatabasePublisher[RouteRow] = {
     logger.debug("select all")
 
-    val q = for {
-      routeRow <- routesTable
-      if routeRow.deletedAt.isEmpty
-    } yield routeRow
-
     db.stream {
-      q.result
+      selectAllQuery.result
```

Comment:
ok. forget everything I just wrote... Found the private lazy val at the bottom of the file... never mind. carry on.